packages/react-storage/src/components/StorageBrowser/context/actions/types.ts

```diff
@@ -0,0 +1,22 @@
+export interface FolderItem {
+  key: string;
+  type: 'FOLDER';
+}
+
+export interface FileItem {
+  key: string;
+  lastModified: Date;
+  size: number;
+  type: 'FILE';
+}
+
+export type LocationItem = FileItem | FolderItem;
+
+export type Permission = 'READ' | 'READWRITE' | 'WRITE';
+export type LocationType = 'OBJECT' | 'PREFIX' | 'BUCKET';
```

Comment:
Will this always be the end part of the full location? Just curious for a prefix type, if you'll want to store the bucket name separately from the prefix.

Reply:
Good question, will think abt it